4 reports.sql

```diff
@@ -0,0 +1,25 @@
+use CostumeDB 
+go 
+
+--1. I need to know which costume is the most popular.
+select top 1 AmountPerCostume = count (*), c.CostumesBought
```

Comment:
You can have a costume which is only in one row in the table, but was the AmountBought can be 12, which would make it the most popular column